gulpfile.js

```diff
@@ -21,7 +21,7 @@ gulp.task('copy', () =>
 
 // Process the Stylus files into CSS
 gulp.task('styles', () =>
-  gulp.src('static/src/styles/**/*.styl')
+  gulp.src('static/src/styles/main.styl')
```

Comment:
Nah, this doesn't affect the watch code (which is further down), a recompile is still triggered if any file is changed.